site/content/en/docs/v0.1/quickstart.md

```diff
@@ -8,22 +8,21 @@ Follow these instructions to install, run, and test etcd with etcd-operator in a
 
 Pre-requisites:
 - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
-- [kustomize](https://github.com/kubernetes-sigs/kustomize)
 - Kubernetes cluster and `kubectl` configured to use it
-  - If you don't have a Kubernetes cluster, you can use [kind](https://kind.sigs.k8s.io/docs/user/quick-start/) to create a local one
+- If you don't have a Kubernetes cluster, you can use [kind](https://kind.sigs.k8s.io/docs/user/quick-start/) to create a local one
 - [cert-manager](https://cert-manager.io/docs/installation/) installed in the cluster
 
 1. Install etcd-operator:
     ```bash
-    kustomize build 'https://github.com/aenix-io/etcd-operator//config/default?ref=main' | kubectl apply -f -
+    kubectl apply -f https://github.com/aenix-io/etcd-operator/releases/download/latest/etcd-operator.yaml
```

Comment:
Probably it should be `v0.1.0` version or something like that. But not latest

Reply:
The `latest` here refers to the latest released version, but the image inside of manifests looks like:

```
        # From my test run
        image: ghcr.io/hiddenmarten/etcd-operator:v0.1.2
```

I don't see any issue here :)